package-lock.json

```diff
@@ -0,0 +1,6999 @@
+{
```

Comment:
you should not have a package-lock.json and a yarn.lock.

remove package-lock.json and only use yarn.

Don't run `npm install`